### ISSUE TITLE
I don't found .env file

### ISSUE THREAD

**Issue by hcamusso** [googlecodelabs/your-first-pwapp#185]
I don't found the .env file for this point:

"Once the repository has been loaded, edit the .env file and update it with your DarkSky API key."

Where is this file or what it its content? Thanks

**Comment by jerrywaller** [comment]
Agreed. This is not made clear anywhere in the documentation (that I've found so far). I'll update if I find it.

**Comment by jerrywaller** [comment]
Ok, it's a hidden dotFile `.env` in the top level of the project directory.

**Comment by jzmntngr** [comment]
> Ok, it's a hidden dotFile `.env` in the top level of the project directory.

how can i unhide it

**Comment by jerrywaller** [comment]
To view it - and edit it - you navigate to the project directory's top level in your file manager of choice (explorer, finder, etc.). In your file manager there is usually an option somewhere to "view hidden files" or something like that. I hope that helps.
